### PR TITLE
Totango #3816

### PR DESCRIPTION
**Pull request by lcaresia**
I don't find a way to do this sources below

### Triggers
* Updated Account
* Updated User

I tried to find/test on Zapier what `Send Data` does, but I don't found nothing.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4071"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

File: components/totango/sources/new-account/new-account.mjs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+      accounts.reverse().forEach(this.emitEvent);
+
+      if (accounts.length < 1000) {
+        return;
+      }
+
+      page++;
+    }
+  },
```
Comment (feyzullah) [inline components/totango/sources/new-account/new-account.mjs:58]: Can these be moved to a common file to reduce code complexity?

File: components/totango/sources/new-user/new-user.mjs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+      users.reverse().forEach(this.emitEvent);
+
+      if (users.length < 1000) {
+        return;
+      }
+
+      page++;
+    }
+  },
```
Comment (feyzullah) [inline components/totango/sources/new-user/new-user.mjs:58]: Can these be moved to a common file to reduce code complexity?

File: components/totango/sources/new-user/new-user.mjs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      });
+
+      users.reverse().forEach(this.emitEvent);
+    },
+  },
+  async run() {
+    let page = 0;
+
+    while (page >= 0) {
+      const users = await this.totango.searchUsers({
```
Comment (feyzullah) [inline components/totango/sources/new-user/new-user.mjs:44]: Would not it be returning all related users when timer ticks? Can we record something to `db` and check new users according to that?

File: components/totango/sources/new-account/new-account.mjs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      });
+
+      accounts.reverse().forEach(this.emitEvent);
+    },
+  },
+  async run() {
+    let page = 0;
+
+    while (page >= 0) {
+      const accounts = await this.totango.searchAccounts({
```
Comment (feyzullah) [inline components/totango/sources/new-account/new-account.mjs:44]: Would not it be returning all related accounts when timer ticks? Can we record something to `db` and check new accounts according to that?

File: components/totango/sources/new-user/new-user.mjs (hunk 3 above)
Comment (lcaresia) [reply components/totango/sources/new-user/new-user.mjs:44]: I don't know what is the best way to work with that, we can save in db and compare, or just emit and the core will compare.

File: components/totango/sources/new-account/new-account.mjs (hunk 4 above)
Comment (lcaresia) [reply components/totango/sources/new-account/new-account.mjs:44]: I don't know what is the best way to work with that, we can save in db and compare, or just emit and the core will compare.

**Review by feyzullah — APPROVED** [review]

**Pull request merged**